specs/2026-04-24_extract-rebuild-recreate-canonical-credentials/tests.md

```
@@ -0,0 +1,481 @@
+    - **Command:** `npm test`
+    - **Expected:** All tests pass
+
+50. `eslint passes`
+    - **Command:** `npx eslint .`
+    - **Expected:** No new warnings or errors
+
+---
+
+## Test Dependency Map
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)